python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -224,6 +224,19 @@ def remove(self):
 
         return
 
+    def get(self, localPath=''):
```

Comment:
the user is not trying to do this. Ganga potentially is when you clean up code to be more readable and to be clearly doing 1 thing. Any special problems to do with the `LocalFile` should be confined to the `LocalFile`.